### PR TITLE
[CoreAudioKit] Add support for Xcode 14 beta 1 & 2

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/coreaudiokit.cs [hunk 1]
```diff
@@ -240,4 +241,18 @@ interface CAInterAppAudioTransportView {
...
 #endif
+
+	[Mac (13,0), iOS (16,0)]
+	[BaseType (typeof(UIViewController))]
+	interface AUGenericViewController
+	{
+
+		[DesignatedInitializer]
+		[Export ("initWithNibName:bundle:")]
+		[PostGet ("NibBundle")]
```
Comment (rolfbjarne) [inline src/coreaudiokit.cs:252]: What's this PostGet for?

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: src/coreaudiokit.cs (hunk 1 above)
Comment (mandel-macaque) [reply src/coreaudiokit.cs:252]: I'm re-exposing the same constructor from the base, I think we can remove the extra attr.

Comment (mandel-macaque) [reply src/coreaudiokit.cs:252]:
```suggestion
```

**Comment by mandel-macaque** [comment]
mono touch tests are unrelated. https://github.com/xamarin/xamarin-macios/pull/15503

**Pull request merged**